en/docs/develop/troubleshooting/logging.md

```
@@ -0,0 +1,186 @@
+
+```ballerina
+log:printInfo("Processing order",
+    orderId = orderId,
+    customerId = customerId,
+    total = total
+);
+```
+
+Output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)